ctgan/synthesizers/tvae.py

```diff
@@ -110,7 +110,9 @@ def fit(self, train_data, discrete_columns=tuple()):
         self.transformer.fit(train_data, discrete_columns)
         train_data = self.transformer.transform(train_data)
         dataset = TensorDataset(torch.from_numpy(train_data.astype('float32')).to(self._device))
-        loader = DataLoader(dataset, batch_size=self.batch_size, shuffle=True, drop_last=True)
+
+        drop_last = True if len(dataset) > 10 * self.batch_size else False
```

Comment:
`variable = True if <condition> else False` seems a bit redundant. You can just do `variable = <condition>` to the same effect:

```
drop_last = len(dataset) > 10 * self.batch_size
```

Also, this change seems to imply that having a last batch which is shorter than the batch size is not a problem. If this is the case, I do not think there is any need to drop it. Let's rather simply skip this line and remove the `drop_last` argument altogether (so it takes the default `False` value).